docs/src/main/asciidoc/hibernate-search-elasticsearch.adoc

```diff
@@ -12,13 +12,9 @@ You have a Hibernate ORM-based application? You want to provide a full-featured
 With this guide, you'll learn how to synchronize your entities to an Elasticsearch cluster in a heart beat with Hibernate Search.
 We will also explore how you can can query your Elasticsearch cluster using the Hibernate Search API.
 
-[WARNING]
-====
-The version of Hibernate Search shipped with Quarkus is still a Beta.
-
-While APIs are quite stable and the code is of production quality and thoroughly tested,
-some features are still missing, performance might not be optimal and some APIs might change before the final release.
-====
+NOTE: This extension is considered `preview`.
+While APIs are quite stable and the code is of production quality and thoroughly tested, some features are still missing, performance might not be optimal and some APIs or configuration properties might change as the extension matures.
+Feedback is welcome on our https://groups.google.com/d/forum/quarkus-dev[mailing list] or as issues in our https://github.com/quarkusio/quarkus/issues[GitHub issue tracker].
```

Comment:
This one is different.

Reply:
Maybe the `Beta` part should remain in the guide, I'm not sure about it.

Reply:
Yes, it's a bit different because the underlying component is in beta.